### ISSUE TITLE
Instance Messenger needs to be fixed up

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7467]
|<img alt="Ben's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Ben reported|
|-|-|
|Trac ID|trac#111|
|Type|task|
|Created|2003-07-21 00:34:34Z|

<details><summary>Searchable metadata</summary>

```
trac-id__111 111
type__task task
reporter__Ben Ben
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__wontfix wontfix
component__words words
keywords__words words
time__1058747674000000 1058747674000000
changetime__1465029897227854 1465029897227854
version__ 
owner__acapnotic acapnotic
cc__glyph cc__radix cc__Ben cc__manman cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Ben's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Ben commented|
|-|-|

```
#!html
<pre>
I am running with Linux 2.4.20, python 2.2.2, twisted
0.99.2, debian testing, tvwm and globally a pretty unstable
system :)

Feel free to contact me for more info. I do not feel pressed
by time, but I am certainly unable to fix this.. without
your help

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Two points:
  - Twisted 0.99.2 is fairly ancient, try 1.0.6 from 
    debian unstable. I presume it will hit testing sometime 
    soon (I think it's being blocked by python2.1-ldaptor, 
    or something?)
  - Instance Messenger is basically unmaintained.  I'm sure
    that if these bugs are fixed, there will be plenty more 
    lurking... if you're fine with that, then that's ok ;)

</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

```
#!html
<pre>
This started out as a Bug posting, entitled:

Instance Messenger IRC Group Conversation bottom text box
does not react after some time and for private conversations
the close button never reacted on my machine

As spiv points out, IM is unmaintained. I'm leaving it in
the release because it _is_ somewhat functional, and I'm
hoping eventually a user will be inspired to hack on it a
little bit. This bug is now a Wishlist item, and I
un-assigned glyph because I know he doesn't have the time to
do this.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="manman's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|manman commented|
|-|-|

What is up

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_8"></a>|@radix commented|
|-|-|

Not much, what is up with you.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_9"></a>|@exarkun set owner to @keturn|
|-|-|

It's tempting to just close this.  I guess there is some actual content in radix's very early comment, though.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun set status to `closed`|
|-|-|

Oops.  We got rid of the im gui in r26801.  So there is no more group conversation bottom text box to not react...

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_11"></a>|@cyli commented|
|-|-|

In changeset 592a4f992eff94aac1518cf06074ad22e10ecdee
```
#!CommitTicketReference repository="" revision="592a4f992eff94aac1518cf06074ad22e10ecdee"
Merge pull request #111 from rodrigc/8395-rodrigc-print

Author: rodrigc
Reviewer: cyli
Fixes: #8395

Convert from print statement to print() function in twisted/words
```
